.github/workflows/full_ci.yml

```diff
@@ -1,9 +1,6 @@
 name: full CI
 
-on:
-  merge_group:
-  pull_request:
-    types: [opened, synchronize, reopened]
+on: [push]
```

Comment:
What is the intent with this change? This commit should be dropped if it is only for testing.

Reply:
I removed this. It was frustrating having to make a PR to get CI to run. Is that a worthwhile change in another PR?

Reply:
Making a PR to get CI to run, as opposed to what exactly? Usually CI is only run on PRs and after merges.